prosper_nn/models/crcnn/crcnn.py

```diff
@@ -174,20 +174,23 @@ def forward(self, Y: torch.Tensor) -> torch.Tensor:
 
         self._check_sizes(Y)
 
+        device = self.init_state_causal.device
```

Comment:
Maybe add to the init that if you pass the init_state_causal, you should be careful about specifing the device. Also, what happens if the user doesn't set it? Is it cpu by default?

Reply:
I changed it that we use the device of CRCNNCell_causal. Than we avoid the problem if the user sets the state. 
Default is CPU